modules/cluster-logging-dashboards-logging.adoc

```diff
@@ -64,8 +64,8 @@ Fetch latency typically takes less time than query latency. If fetch latency is
 |FluentD emit count
 |The total number of Fluentd messages per second for the Fluentd default output, and the retry count for the default output.
 
-|FluentD Buffer Availability
-|The percent of the Fluentd buffer that is available for chunks. A full buffer might indicate that Fluentd is not able to process the number of logs received.
+|FluentD Buffer Usage
+|The percent of the Fluentd buffer that is using for chunks. A full buffer might indicate that Fluentd is not able to process the number of logs received.
```

Comment:
Could we rephrase as 'The percent of the Fluentd buffer that is being used for chunks'?